sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -366,15 +371,19 @@ _FT_PROVISIONING_HARNESS = "//sw/host/provisioning/ft"
         },
         fpga = fpga_params(
             timeout = "long",
+            assemble = "{ft_personalize}@{slot_a} {rom_ext}@{slot_b}",
             binaries =
                 {
                     ":sram_ft_individualize_{}".format(config["otp"]): "sram_ft_individualize",
                     ":ft_personalize_{}".format(sku): "ft_personalize",
+                    "//sw/device/silicon_creator/rom_ext/sival:rom_ext_fake_prod_signed_slot_b": "rom_ext",
```

Comment:
I think this actually broke the `//sw/device/silicon_creator/manuf/base:ft_provision_prodc_fpga_hyper310_rom_with_fake_keys` target, because this hardcodes the `sival` ROM_EXT. But no worries, #24971 will fix it again. 

It also looks like the `ft_provision_*` flows are not being run in CI b/c they are marked as `long` tests... I will fix that in a follow up too.

Reply:
#24971 has been merged now so everything should be fixed.

Reply:
#24975 will ensure the FT flow runs in CI now too.